app/assets/config/manifest.js

```diff
@@ -4,6 +4,7 @@
 
 //= link components/_taxon-list.css
 //= link components/_topic-list.css
+//= link components/_intervention.css
```

Comment:
We can remove this line if the approach mentioned in the show.html.erb view template is used